src/overlays/actors/ovl_En_Guest/z_en_guest.h

```diff
@@ -8,7 +8,19 @@ struct EnGuest;
 
 typedef struct EnGuest {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1C4];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[16];
+    /* 0x01F0 */ Vec3s transitionDrawTable[16];
+    /* 0x0250 */ void (*actionFunc)(struct EnGuest* this, GlobalContext* globalCtx);
+    /* 0x0254 */ ColliderCylinder colliderCylinder;
```

Comment:
our convention is to call colliders just "collider" unless the actor has more than one